app/core/utils.js

```diff
@@ -67,23 +67,21 @@ utils.getIn = (obj, props, notFound) => {
 
 utils.isAcceptedBrowser = () => {
   var userAgent = navigator.userAgent.toLowerCase();
-  return (userAgent.indexOf('chrome') > -1 
-    || userAgent.indexOf('firefox') > -1 
-    || userAgent.indexOf('crios') > -1 
+  return (userAgent.indexOf('chrome') > -1
+    || userAgent.indexOf('firefox') > -1
+    || userAgent.indexOf('crios') > -1
     || userAgent.indexOf('edge') > -1);
 };
 
```

Comment:
See my other comment this function is useful.
```suggestion

utils.isChrome = () => {	
  var userAgent = navigator.userAgent.toLowerCase();	
  return (userAgent.indexOf('chrome') > -1 	
    || userAgent.indexOf('crios') > -1);	
};

```